src/huggingface_hub/inference/_common.py

```diff
@@ -75,9 +75,15 @@ class ModelStatus:
 
     Args:
         loaded (`bool`):
-            If the model is currently loaded.
+            If the model is currently loaded into Hugging Face's infrastructure. Models
+            are loaded on-demand to avoid waste, and when loaded they can be use.
         state (`str`):
-            The current state of the model. This can be 'Loaded', 'Loadable', 'TooBig'
+            The current state of the model. This can be 'Loaded', 'Loadable', 'TooBig'.
+            If a model's state is 'Loadable' (meaning it's not too big and has a
+            supported backend) and it's desired to be 'Loaded', the procedure is:
+            1. Make a request to a Hugging Face endpoint can be made to load the model.
```

Comment:
@Wauplin can you share a link to docs on the exact endpoint and JSON schema to use?

Reply:
There was a confusion I think. See https://github.com/huggingface/huggingface_hub/pull/1740#pullrequestreview-1679325742 and https://github.com/huggingface/huggingface_hub/pull/1740#discussion_r1360274608 :)